/*
 * This file is part of the VSS-Vision project.
 *
 * This Source Code Form is subject to the terms of the GNU GENERAL PUBLIC LICENSE,
 * v. 3.0. If a copy of the GPL was not distributed with this
 * file, You can obtain one at http://www.gnu.org/licenses/gpl-3.0/.
 */

package vss_command;

message Robot_Command{
	required uint32	id = 1;
	required float left_vel = 2;
	required float right_vel = 3;
}

message Global_Commands{
	optional uint32			id = 1;
	required bool			is_team_yellow = 2;
	repeated Robot_Command	robot_commands = 3;
}

